### PR TITLE
Replace all instances of rawgit with Raw.githack (Issue #2308)

### LINKED ISSUE #2308: Rawgit inaccurately mentioned in Steps 3 and 6
### Problem
Rawgit is mentioned multiple times in the tutorial even though the website is no longer supported and raw.githack is used elsewhere in the tutorial instead.

### Steps to reproduce the problem
Simply go to steps 3 and 6 of the tutorial and search for rawgit. The issue is not present in other tutorial steps.

### Screenshots
Screen Shot 2019-05-27 at 11 38 17 AM
Screen Shot 2019-05-27 at 11 39 15 AM
Screen Shot 2019-05-27 at 11 40 11 AM
Screen Shot 2019-05-27 at 11 40 41 AM

### Proposed solution
I would propose simply replacing all instances of rawgit with raw.githack.

### PR DESCRIPTION
**Pull request by SNutakki**
Fixes #2308

### Description
All instances of rawgit are replaced by Raw.githack.

### RawGit preview link

https://raw.githack.com/SNutakki/SNutakki.github.io/rawgit/#!pages/vi/vi-github-and-markdown.md
https://raw.githack.com/SNutakki/SNutakki.github.io/rawgit/#!pages/vi/vi-github-issues.md

### CODE REVIEW

**Comment by jeevast** [comment]
@SNutakki  I was facing that issue with "rawgit" before and figured out myself, nice that you're looking forward to fix it.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,5 +12,5 @@ Fixes #
 ### Description
 
 
-### RawGit preview link
-<!-- rawgit link to page(s) changed -->
+### Raw.githack preview link
+<!-- Raw.githack link to page(s) changed -->
```
Comment (lmmrssa) [inline .github/PULL_REQUEST_TEMPLATE.md]: can you convert `Raw.githack` to `raw.githack`

File: pages/vi/vi-github-and-markdown.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ Now, check what this looks like on your own page `https://raw.githack.com/YourUs
 
 **NOTE**:
 • If you don't see any changes in your page after editing your file then you need to clear your browser's cache or open your page in your browser's **"incognito"** or **"privacy"** mode. You can also press Ctrl+Shift+R or Ctrl+F5 to reload the page with cache cleared.
-• As MDwiki site is what we use for "production," please **always check** if everything renders as you expected using rawgit link. There are [different flavors of Markdown](https://github.com/commonmark/CommonMark/wiki/Markdown-Flavors). Use GitHub's preview tab for reference only.
+• As MDwiki site is what we use for "production," please **always check** if everything renders as you expected using Raw.githack link. There are [different flavors of Markdown](https://github.com/commonmark/CommonMark/wiki/Markdown-Flavors). Use GitHub's preview tab for reference only.
```
Comment (lmmrssa) [inline pages/vi/vi-github-and-markdown.md]: can you convert `Raw.githack` to `raw.githack`

File: pages/vi/vi-github-and-markdown.md [hunk 3]
```diff
@@ -126,7 +126,7 @@ Once you have your profile ready, it's time to create a pull request. Click on o
 **There are a few things to watch out before clicking on the green "Create pull request" button**. Make sure you:
 
 * give the pull request a short and descriptive title (e.g. create YourUserName.md)
-* include the rawgit link to your Markdown profile page in the pull request description
+* include the Raw.githack link to your Markdown profile page in the pull request description
```
Comment (lmmrssa) [inline pages/vi/vi-github-and-markdown.md]: can you convert `Raw.githack` to `raw.githack`

File: pages/vi/vi-github-issues.md [hunk 4]
```diff
@@ -78,7 +78,7 @@ For edits to this markdown wiki, we recommend that your commit messages only con
 * Use the imperative mood in the subject line
 
 Here is an example of how you would write your commit message:
-`$ git commit -m "Add commit message style guide and add Rawgit section (fixes #841)"`
+`$ git commit -m "Add commit message style guide and add Raw.githack section (fixes #841)"`
```
Comment (lmmrssa) [inline pages/vi/vi-github-issues.md]: can you convert `Raw.githack` to `raw.githack`

File: pages/vi/vi-github-issues.md [hunk 5]
```diff
@@ -108,7 +108,7 @@ If you login to your own fork of OLE on Github, Github will notice that you rece
...
+For everyone's convenience, we highly recommend that you follow a certain structure when submitting a pull request. We suggest to include the issue for which the pull request is resolving (for example, if your pull request fixes the issue number 320, then add `#320` to your pull request) to the title of your PR and also the Raw.githack link into the description of the pull request. This saves a lot of time for the person reading your pull request as he/she does not need to search and match the issues with the corresponding PR and is able to instantly see the changes pushed via the Raw.githack link.
```
Comment (lmmrssa) [inline pages/vi/vi-github-issues.md]: can you convert `Raw.githack` to `raw.githack`

File: pages/vi/vi-github-issues.md [hunk 6]
```diff
@@ -118,7 +118,7 @@ Additionally, creating a checklist will help others understand what you have don
 [x] Are there any unneeded files in the pull request?
 [x] Did they make a branch for their patch?
 [x] Does the pull request actually fix the issue?
-[x] Check the pull request on rawgit, does it display without any errors?
+[x] Check the pull request on Raw.githack, does it display without any errors?
```
Comment (lmmrssa) [inline pages/vi/vi-github-issues.md]: can you convert `Raw.githack` to `raw.githack`

**Pull request merged**